autoemulate/experimental/compare.py

```diff
@@ -307,7 +326,7 @@ def plot(  # noqa: PLR0912, PLR0915
         if isinstance(y_pred, DistributionLike):
             y_variance = y_pred.variance
```

Comment:
Should we update the plot function and the `plot_xy` called within passing in `r2_test_std` instead of `y_variance`? I set it up previously to calculate the std within that function:
```
if y_variance is not None:
        y_variance_sorted = y_variance[sort_idx]
        y_std = np.sqrt(y_variance_sorted)
```

Reply:
Yeah I think we can keep the `plot_xy` as is `y_std` as implemented. Though it might be interesting to add plot functionality comparing the `R^2` across models with error bars for the $R^2$ to visually compare the models too?